src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -1,18 +1,32 @@
 import * as React from 'react'
-import { Dropdown, DropdownProps, Popup } from 'decentraland-ui'
+import { Color4 } from 'decentraland-ecs'
+import { Dropdown, DropdownProps, DropdownItemProps, Popup, Icon } from 'decentraland-ui'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import ViewPort from 'components/ViewPort'
 import { AvatarAnimation, PreviewType } from 'modules/editor/types'
+import { getSkinColors, getEyeColors, getHairColors } from 'modules/editor/utils'
 import { WearableBodyShape } from 'modules/item/types'
 import { Props } from './CenterPanel.types'
 import './CenterPanel.css'
 
-export default class CenterPanel extends React.PureComponent<Props> {
+type State = {
```

Comment:
Should we need to move this to `CenterPanel.types.ts`?

Reply:
Yes! slipped thru the cracks